attributes/default.rb

```diff
@@ -59,20 +59,27 @@
 case node['platform_family']
 
 when 'rhel'
-  # RHEL5 edge case package set and portmap name
-  if node['platform_version'].to_i <= 5
-    default['nfs']['packages'] = %w(nfs-utils portmap)
-    default['nfs']['service']['portmap'] = 'portmap'
-  elsif node['platform_version'].to_i >= 7
-    default['nfs']['service']['lock'] = 'nfs-lock'
-    default['nfs']['service']['server'] = 'nfs-server'
-    default['nfs']['service']['idmap'] = 'nfs-idmap'
-
-    if node['platform_version'] == '7.0.1406'
-      default['nfs']['client-services'] = %w(nfs-lock.service)
+  case node['platform']
+    when 'amazon'
```

Comment:
They're valid for 2014.09 and 2015.03 which are the latest versions, so maybe you can just specify supports amazon ~> 2014.09.

Both versions started as fork of CentOS 6 so actually you can just use the default names for the services outside of the platform_family = 'rhel' block.

If the next version of amazon (2015.09, which will be out around October) forks from Rhel 7 then it would use the rhel 7 values, otherwise no change.